ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -53,6 +53,7 @@ jobs:
     job:
       class: Build
       args:
+        copr: '@pki/master'
```

Comment:
We're seeing more than a few very strange failures. The pki/master copr repo is 11.6.0. I wonder if we should instead test against 11.5.0 which is what is in F40+ right now. My freeipa copr repo has a 11.5.0 build in it.